.travis.yml

```diff
@@ -1,6 +1,21 @@
-language: node_js
-node_js:
-  - "8.9.4"
+sudo: required
+
+services:
+  - docker
+  
+dist: trusty
```

Comment:
Can this be xenial? We use 16.04 everywhere so would prefer to be consistent here too.